php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -23,6 +23,12 @@ namespace Loris\Behavioural;
  */
 class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
 {
+    /**
+     * FIXME: LINST instruments should parse the full name
+     * and the subtest names from the linst file, not select
+     * them from the database.
+     */
+    use \LORIS\LegacyInstrumentTrait;
```

Comment:
I think maybe you don't need this namespacing because it's in `php/libraries` not in `src/`?

Reply:
It's not necessary, but I thought it would be useful.. maybe it's conflicting because the same namespace is used for the `src` directory, I'll remove it to see